shared/common-adapters/kb-text-input.native.tsx

```diff
@@ -24,8 +24,9 @@ if (isAndroid) {
 
       // @ts-ignore no RN types
       props.style = [this.props.style]
-      props.autoCapitalize =
-        UIManager.AndroidTextInput.Constants.AutoCapitalizationType[props.autoCapitalize || 'sentences']
+      props.autoCapitalize = UIManager.getViewManagerConfig(
```

Comment:
Got a yellow box that said we should be using this

Reply:
nice :+1: I don't think this was supported in 0.58